nx-X11/include/Xos_r.h

```diff
@@ -476,15 +456,9 @@ typedef struct {
     (sp).sptr )
 
 #elif defined(XUSE_NETDB_R_API)
-/*
- * POSIX does not specify _r equivalents for <netdb.h> API, but some
- * vendors provide them anyway.  Use them only when explicitly asked.
- */
```

Comment:
Uh, no, don't drop `_POSIX_REENTRANT_FUNCTIONS` *everywhere*, you need to revert this! I **only** meant deleting the 7 lines below.